cosmwasm/deploy-contract.js

```diff
@@ -0,0 +1,404 @@
+'use strict';
+
+require('dotenv').config();
+
+const { SigningCosmWasmClient } = require('@cosmjs/cosmwasm-stargate');
+const { DirectSecp256k1HdWallet } = require('@cosmjs/proto-signing');
+
+const { printInfo, loadConfig, saveConfig, isString, isStringArray, isNumber } = require('../evm/utils');
+const { uploadContract, instantiateContract } = require('./utils');
+
+const { Command, Option } = require('commander');
+
+const makeServiceRegistryInstantiateMsg = ({ governanceAccount }) => {
+    if (!isString(governanceAccount)) {
```

Comment:
isn't there a function to specifically validate cosmwasm addresses somewhere in `cosmjs` ? same question for the rest of the functions